### PR TITLE
Add "missing payload" check to crasher

### PR DESCRIPTION
**Pull request by ffuugoo**
Trying to catch that elusive bug on chaos-testing, when some points seem to miss payload after restart. 🥲

### CODE REVIEW

File: src/client.rs [hunk 1]
```diff
@@ -335,11 +336,16 @@ pub async fn insert_points_batch(
...
+            let mut payload = random_payload(Some(payload_count));
+
+            if timestamp_payload {
+                payload.insert(
+                    "timestamp",
+                    chrono::Utc::now().to_rfc3339_opts(chrono::SecondsFormat::Millis, true),
+                );
+            }
+
+            points.push(PointStruct::new(point_id, vectors, payload));
```
Comment (agourlay) [inline src/client.rs:348]: Is it possible to conserve the current behavior of not adding automatically a payload on insertion if the new CLI option is not set?

Comment (ffuugoo) [reply src/client.rs:348]: But the behavior is exactly the same as it was before, if the option is not set.

E.g., I've changed this:

```rust
            points.push(PointStruct::new(
                point_id,
                vectors,
                random_payload(Some(payload_count)),
            ));
```

To this:
```rust
            let mut payload = random_payload(Some(payload_count));

            if timestamp_payload {
                payload.insert(
                    "timestamp",
                    chrono::Utc::now().to_rfc3339_opts(chrono::SecondsFormat::Millis, true),
                );
            }

            points.push(PointStruct::new(point_id, vectors, payload));
```

But `payload` was initialized to `random_payload(Some(payload_count))` even before my change.

It's just when workload calls `insert_points_batch`, it pass `payload_count: 0`, and (I guess), this produces an empty payload.

Comment (agourlay) [reply src/client.rs:348]: Thanks a lot for the explanation, I forgot about the zero trick :sweat_smile:

**Review by agourlay — APPROVED** [review]
Let's give this a try :+1:

**Pull request merged**